Recrutify/Recrutify.DataAccess/Repositories/ProjectRepository.cs

```diff
@@ -24,5 +24,13 @@ public Task<IEnumerable<ProjectPrimarySkill>> GetPrimarySkills(Guid id)
              .Project(p => p.PrimarySkills)
              .FirstOrDefaultAsync();
         }
+
+        public Task UpdateCurrentApplicationsCount(Guid id)
+        {
+            var filter = _filterBuilder.Eq(x => x.Id, id);
+            var updateBuilder = Builders<Project>.Update;
```

Comment:
you can not write separate var updateBuilder, you use it once, so can just write Builders<Project>.Update.... in next line

Reply:
done